csvcubed/tests/behaviour/inspectcli.feature

```diff
@@ -0,0 +1,11 @@
+# Feature: Testing inspect cli (WIP).
+
+#     Scenario: inspect cli should produce printable for a valid dataset json-ld input
+#         Given the existing test-case file "cli/inspect/dataset.csv-metadata.json"
```

Comment:
Once we're more comfortable with what the printed structure will looks like, it'll be good to get some behave tests working for it.

Reply:
I agree - I have this as an item in my list.